commons-geometry-euclidean/src/main/java/org/apache/commons/geometry/euclidean/threed/Line.java

```diff
@@ -236,5 +238,29 @@ public Vector3D intersection(final Line line) {
     public SubLine wholeLine() {
         return new SubLine(this, new IntervalsSet(precision));
     }
+    
+    
 
+    @Override
+    public int hashCode() {
+        throw new IllegalStateException("Must not be used in maps.");
```

Comment:
We should provide an actual implementation of this method, especially since `equals()` is overridden.

Reply:
done